images/rpmbuild-centos-stream9/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ghcr.io/geonet/base-images/centos:stream8
+# Install prerequisites
+RUN dnf module enable -y nodejs:16 \
```

Comment:
nodejs 16 is the default for el9 so this line is not needed.

Reply:
Good catch. Must've pasted in the same as stream8. Updating now